interface/client/styles/popupWindows.import.less

```diff
@@ -9,6 +9,28 @@
         color: @colorLinkFocus;
     }
 
+    &.update-available {
+        .text {
+            text-align: left;
+        }
+
+        h1 {
+            font-size: 200%;
+            margin-bottom: 20px;
+
+            &.no-upate {
+                font-size: 180%;
+            }
+        }
+
+        #image {
```

Comment:
I left the ids as they were because I figured there was probably a good reason for them being there. Have now removed them.